browser/ui/tabs/split_view_tab_strip_model_adapter.cc

```diff
@@ -282,6 +282,10 @@ void SplitViewTabStripModelAdapter::TabGroupedStateChanged(
     std::optional<tab_groups::TabGroupId> group,
     content::WebContents* contents,
     int index) {
+  if (!group.has_value()) {
```

Comment:
I don't think this is what we want. When |group| is null, it means a tab was ungrouped and we also want the other tab ungrouped than. You can see that we're checking |group| in `SynchronizeGroupedState()`, which is posted at the end of this method. What do you think?

Reply:
Okay, As I understand that null group is intended, Will add a check like |ContainsIndex(index)| beforehand with TODO to fix the crash now.